core/src/main/scala/com/rallyhealth/vapors/core/data/NamedLens.scala

```diff
@@ -1,26 +1,49 @@
 package com.rallyhealth.vapors.core.data
 
+import cats.data.NonEmptySet
+import cats.kernel.Semigroup
 import com.rallyhealth.vapors.core.macros.NamedLensMacros
 
-case class NamedLens[A, B](
-  path: DataPath,
-  get: A => B,
-) {
+import scala.collection.Factory
 
-  def compose[C](lens: NamedLens[C, A]): NamedLens[C, B] = {
-    copy(
-      path = lens.path ::: this.path,
-      get = get.compose(lens.get),
-    )
+object NamedLens {
+
+  type Id[A] = NamedLens[A, A]
+  val Id: NamedLens[Any, Any] = NamedLens(DataPath(Nil), identity[Any])
+  def id[A]: NamedLens[A, A] = Id.asInstanceOf[Id[A]]
+
+  type Fn[A, B] = NamedLens.Id[A] => NamedLens[A, B]
+  type ==>[A, B] = NamedLens.Id[A] => NamedLens[A, B]
```

Comment:
I don't see any use of `==>` in the codebase, so maybe this can be removed?  Should `Fn` be more descriptive?

Reply:
Oh, good catch. This was an experiment that I didn't clean up. I'll remove these.

Reply:
You are actually using `Fn`--I've added this back and pushed it to kick off another compile.